admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
+
+New option to granularly configure public link password enforcement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With ownCloud 10 the *"File Drop"* feature has been merged to public link permissions. As this kind of public link does not give recipients access to any contents but just gives them the possibility to "drop" files it might not be desirable for some scenarios to enforce password protection for such shares while other public links should require setting a password. For this password enforcement for public links has been split to apply based on permissions given (*read-only, read & write, upload only/File Drop*). Please check the administration settings *'Sharing'* section and configure as desired.
+
+New option to exclude apps from integrity check
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By verifying signature files the *integrity check* ensures that the code running in an ownCloud instance has not been altered by third parties. Naturally this check can only be successful for code that has been obtained from official ownCloud sources. In the case of external modifications (e.g., for theming) the *integrity check* will fail and notify the administrator. For intended modifications within apps these can now be excluded from the *integrity check* by using the *config.php* option ``'integrity.ignore.missing.app.signature' => ['app1', 'app2', 'app3'],``. See *config.sample.php* for more information.
+
+New occ command to modify user details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is now possible to modify user details like display names or mail addresses via the command ``occ user:modify``. Please append ``--help`` for more information.
+
+occ files:scan can now be executed for groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Apart from using the ``occ files:scan`` command for *single users* and *whole instances* it can now be executed for *groups* using ``-g, --groups=GROUPS``. Please append ``--help`` for more information.
+
+New configurable default format for syslog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When using syslog as the log type (``'log_type' => 'syslog',`` in *config.php*) the default format has been changed to include *request IDs* for easier debugging. Additionally the log format has been made configurable using ``'log.syslog.format'`` in *config.php*. If you require a certain log format to forward to external applications, please check the new format and *config.sample.php* on how to change it.
+
+New config option to enable fallback to HTTP for federated shares
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For security reasons federated sharing (sharing between different ownCloud instances) strictly requires HTTPS (SSL/TLS). When this behavior is undesired the insecure fallback to HTTP needs to be enabled explicitly by setting ``'sharing.federation.allowHttpFallback' => false,`` to ``true`` in *config.php*.
```

Comment:
Additional clarification: the fallback works as follows: first this OC server tries to connect to a remote with HTTPS and if that fails it would fall back to HTTP. With that option disabled it just gives up after a failed HTTPS call.